src/WeSay.ConfigTool/Tasks/TaskListPresentationModel.cs

```diff
@@ -43,6 +45,19 @@ private IEnumerable<ICareThatWritingSystemIdChanged> ICareThatWritingSystemIdCha
 			}
 		}
 
+		private IEnumerable<ICareThatMeaningFieldChanged> ICareThatMeaningFieldChangedTasks
+		{
+			get
+			{
+				foreach (object task in Tasks)
+				{
+					if (null == task as ICareThatMeaningFieldChanged)
```

Comment:
I would use the LINQ function `OfType<ICareThatMeaningFieldChanged>()` here. This could be simplified to
```
return Tasks.OfType<ICareThatMeaningFieldChanged>();
```